examples/BasicProvider/BasicProvider.fsproj

```diff
@@ -1,38 +0,0 @@
-﻿<Project>
-  <!-- See https://stackoverflow.com/questions/43921992/how-can-i-use-beforebuild-and-afterbuild-targets-with-visual-studio-2017 -->
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
-  <Import Project="..\..\netfx.props" />
-
-  <PropertyGroup>
-     <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
-     <!-- Use an explicit nuspec to allow us to specify "references", the metadata properties at https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#nuget-metadata-properties don't allow this --> 
-     <NuspecFile>BasicProvider.nuspec</NuspecFile>
-     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-     <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
-  </PropertyGroup>
-
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
-
-  <ItemGroup>
-    <Compile Include="BasicProvider.Runtime.fs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <!-- Be explicit about the version of FSharp.Core since we mention it in the manual nuget package spec -->
-    <PackageReference Include="FSharp.Core" Version="4.3.4" />
-  </ItemGroup>
-
-  <Target Name="AfterBuild">  
```

Comment:
This AfterBuild is necessary to place the TPDTC in the right location with respect to the TPRTC outputs.

Reply:
e.g. 

```
    <!-- We place the TPDTC DLLs in a well-known locations with respect to the runtime DLL -->
    <Copy SourceFiles="..\BasicProvider.DesignTime\bin\$(Configuration)\netstandard2.0\BasicProvider.DesignTime.dll" DestinationFolder="$(OutputPath)\typeproviders\fsharp41\netstandard2.0" />
    <Copy SourceFiles="..\BasicProvider.DesignTime\bin\$(Configuration)\netstandard2.0\FSharp.TypeProviders.SDK.dll" DestinationFolder="$(OutputPath)\typeproviders\fsharp41\netstandard2.0" />
```

Reply:
It is not, see other comments

Reply:
Documented in https://github.com/fsprojects/FSharp.TypeProviders.SDK/pull/339#issuecomment-716881125.